src/ai/backend/manager/models/image.py

```diff
@@ -1034,7 +1036,11 @@ async def _in_user_scope(
 
         for row in await self.db_session.scalars(_img_query_stmt):
             image_row = cast(ImageRow, row)
-            if not self._is_image_accessible_for_user(image_row, allowed_registries, scope.user_id):
+            if image_row.labels.get(
+                "ai.backend.customized-image.owner"
+            ) is None or not self._is_image_accessible_for_user(
```

Comment:
Wouldn't `image_row.is_customized_image()` be more readable than `image_row.labels.get("ai.backend.customized-image.owner")`?